multicluster/controllers/multicluster/member/serviceexport_controller.go

```diff
@@ -402,6 +423,12 @@ func (r *ServiceExportReconciler) updateSvcExportStatus(ctx context.Context, req
 	case serviceNotFound:
 		newCondition.Reason = getStringPointer("service_not_found")
 		newCondition.Message = getStringPointer("the Service does not exist")
+	case serviceNotSupported:
+		newCondition.Reason = getStringPointer("service_type_not_supported")
+		newCondition.Message = getStringPointer("the ExternalName type of Service is not supported")
+	case serviceClusterIPEmpty:
+		newCondition.Reason = getStringPointer("service_clusterip_empty")
+		newCondition.Message = getStringPointer("the ClusterIP of Service is empty")
 	case serviceWithoutEndpoints:
 		newCondition.Reason = getStringPointer("service_without_endpoints")
 		newCondition.Message = getStringPointer("the Service has no Endpoints, failed to export")
```

Comment:
Just say: "Service has no Endpoints"

And for the following two:

"The Service is an imported Service and cannot be re-exported"

"Service is successfully exported". But for `True`, do we still need a reason and message? I believe no.

Reply:
Done, removed reason and message if succeed.